src/test/java/org/candlepin/resource/ConsumerResourceTest.java

```diff
@@ -876,7 +876,7 @@ public void validIdCertDoesNotRegenerate() {
         ComplianceStatus status = new ComplianceStatus();
         when(complianceRules.getStatus(any(Consumer.class), any(Date.class), anyBoolean()))
             .thenReturn(status);
-        consumer.setIdCert(createIdCert(TestUtil.createDate(2025, 6, 9)));
+        consumer.setIdCert(createIdCert(TestUtil.createDateOffset(1, 0, 0)));
```

Comment:
This test started failing today (03/11) because this hard coded date is exactly 90 days from now and we consider ID certs expired if the expiration date is 90 days in the future. So, this test began taking a different code path because the ID cert was considered expired. Updated to use a relative date.

Reply:
Given the other branches got a dedicated PR for this, that's probably apt here as well; if for no other reason than git blame and history cleanliness.

Reply:
I rebased with main to pick up the changes that were merged into main that addressed this issue